Build.fsx

```diff
@@ -115,14 +116,14 @@ Target "TestOnly" (fun _ ->
     let nunit2TestAssemblies = !! (sprintf "Src/AutoFixture.NUnit2.*Test/bin/%s/*Test.dll" configuration)
 
     nunit2TestAssemblies
-    |> NUnit (fun p -> { p with StopOnError = false
-                                OutputFile = "NUnit2TestResult.xml" })
+    |> NUnitSequential.NUnit (fun p -> { p with StopOnError = false
+                                                OutputFile = "NUnit2TestResult.xml" })
```

Comment:
It's required to make code compilable - we have names collision after open of the `Fake.AppVeyor` namespace. 

Given that this change is caused by other changes related to this commit, do you agree that it's OK to keep it as is? :)